player/tests.py

```diff
@@ -47,43 +60,50 @@ def test_delete_user_deletes_player(self, saved_player):
         with pytest.raises(Player.DoesNotExist):
             Player.objects.get(pk=saved_player.pk)
 
-    def test_create_player_with_valid_fields(self, user):
-        Player.objects.create(
-            user=user,
+    def test_create_player_with_valid_fields(self):
+        Player.create(
+            username="daniel",
+            password="password",
             birth_date='1990-01-01',
             favorite_ball_game=BallGame.Soccer).full_clean()
 
-    def test_create_player_with_invalid_birth_date(self, user):
+    def test_create_player_with_invalid_birth_date(self):
         with pytest.raises(ValidationError):
-            Player.objects.create(
-                user=user,
+            Player.create(
+                username="daniel",
+                password="password",
                 birth_date='invalid date',
                 favorite_ball_game=BallGame.Soccer).full_clean()
 
-    def test_create_player_with_invalid_favorite_ball_game(self, user):
+    def test_create_player_with_invalid_favorite_ball_game(self):
         with pytest.raises(ValidationError):
-            Player.objects.create(
-                user=user,
+            Player.create(
+                username="daniel",
+                password="password",
                 birth_date='1990-01-01',
                 favorite_ball_game='invalid game').full_clean()
 
-    def test_create_player_with_blank_birth_date(self, user):
+    def test_create_player_with_blank_birth_date(self):
         with pytest.raises(ValidationError):
-            Player.objects.create(
-                user=user,
+            Player.create(
+                username="daniel",
+                password="password",
                 birth_date='',
                 favorite_ball_game=BallGame.Soccer).full_clean()
 
-    def test_create_player_with_blank_favorite_ball_game(self, user):
+    def test_create_player_with_blank_favorite_ball_game(self):
         with pytest.raises(ValidationError):
-            Player.objects.create(
-                user=user,
+            Player.create(
+                username="daniel",
+                password="password",
                 birth_date='1990-01-01',
                 favorite_ball_game='').full_clean()
 
-    def test_create_player_with_null_favorite_ball_game(self, user):
-        with pytest.raises(ValidationError):
-            Player.objects.create(
-                user=user,
+    def test_create_player_with_null_favorite_ball_game(self):
+        with pytest.raises(IntegrityError):
```

Comment:
As I see, the purpose of the test is to raise error when you put
None value on favorite_ball_game field.
You don't need the ValidationError here?
As I know, ValidtionError is for that case or maybe I wrong?